Algorithms/is_unique.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# CtCI 6th Edition Problem 1.1
+# Implement an algorithm to determine if a string has all unique characters.
+
+
+def is_unique(string):
+    chars = set(string)
```

Comment:
This is excellent feedback and really so obvious when I think about it. I'm surprised as several of the SO discussions I reviewed had similar tacts with creating a set.